full/src/test/java/apoc/load/LoadHtmlTest.java

```diff
@@ -233,6 +233,20 @@ public void testQueryWithFailsSilentlyWithListWithExceptionIfIncorrectCharset()
         testIncorrectCharset("CALL apoc.load.html('" + VALID_PATH + "',{a:'a'}, {failSilently: 'WITH_LIST', charset: '" + INVALID_CHARSET + "'})");
     }
 
+    @Test(expected = QueryExecutionException.class)
+    public void testFailsWithIncorrectBrowser() {
```

Comment:
why not use the `ExpectedException`Junit 4 Rule? This would make the flow much easier to (write and) read?